### ISSUE TITLE
Date format is not working with where method 

### ISSUE THREAD

**Issue by kbtonmoy** [eugenezadorin/airtable-php#2]
I have a column that contains dates but I can't fetch the arrays based on the date format. 

`$dataFDU = $client->table($tableName)
            ->select('*') 
            ->where(["Deadline" => "2022-04-02"])
            ->execute();`

Here is my workaround which returns an empty array. Some help will be appreciated.

**Comment by eugenezadorin** [comment]
Hi! Sorry for the long wait. Your query actually converts into formula like this: `{Deadline}='2022-04-02'`.

Looks like Airtable don't support such comparisons and we have to use datetime functions to compare values properly.

I found following workaround for you:

```php
$dataFDU = $client->table($tableName)
    ->select('*')
    ->whereRaw("IS_SAME(Deadline, '2022-04-02')") // using raw formula here
    ->execute();
```

It's temporary solution, but I will do some research and probably implement something more convenient, like that:

```php
$query->whereDate('Deadline', '2022-04-02');
```

**Comment by eugenezadorin** [comment]
Implemented in [v0.4.0](https://github.com/eugenezadorin/airtable-php/releases/tag/v0.4.0)

Please check out documentation - https://github.com/eugenezadorin/airtable-php#date-filtering
